### ISSUE TITLE
requirements.txt

### ISSUE THREAD

**Issue by vangoghvapor** [susanli2016/NLP-with-Python#35]
I'm attempting to follow the steps and running into issues with this line

"    simple_rnn_model = simple_model(tmp_x.shape, max_french_sequence_length, english_vocab_size, french_vocab_size)  "

this seems to be a version difference in our tensorflow/keras versions. Could you potentially create a requirements.txt and include version numbers of these packages?

**Comment by vangoghvapor** [comment]
when running "    tests.test_simple_model(simple_model)" I'm getting an attribute error saying the object doesn't contain loss functions
